ui/src/app/pages/workspace/workspace-library.spec.tsx

```diff
@@ -85,4 +85,17 @@ describe('WorkspaceLibrary', () => {
     expect(cardNameList).toEqual([publishedWorkspaceStubs[2].name]);
   });
 
+  it('should have tutorial workspaces as default tab', async () => {
```

Comment:
the code change looks good.  I don't really understand this test.

Reply:
Basically if you do not click any tab and launch the component, TUTORIAL workspace is the one that should start showing up.